clients/open-data-soft/clients/annuaire-service-public/index.ts

```diff
@@ -71,7 +71,7 @@ const clientAnnuaireServicePublicByName = async (
       .toUpperCase()
       // Remove accents
       .normalize('NFD')
-      .replace(/\p{Diacritic}/gu, '');
+      .replace(/[\u0300-\u036f]/g, '');
     return nom === name;
```

Comment:
Could you maybe mutualize with https://github.com/annuaire-entreprises-data-gouv-fr/site/blob/f6614e0324f718aadbdda9308d5fc2f8d0dc16f4/utils/helpers/formatting/formatting.ts#L222